Configurations/unix-Makefile.tmpl

```diff
@@ -679,9 +679,12 @@ tags TAGS: FORCE
 
 # Release targets (note: only available on Unix) #####################
 
+# If your tar command doesn't support --owner and --group, make sure to
+# use one that does, for example GNU tar
 TAR_COMMAND=$(TAR) $(TARFLAGS) --owner 0 --group 0 -cvf -
```

Comment:
It wasn't my intention to suggest TAR_COMMAND rename. [I suppose I should have written "which should really *have been* called"?] The implication was rather to detect if tar (or more specifically $(TAR)) is GNU and do something about *that*...